content/zh-cn/docs/concepts/security/pod-security-standards.md

```diff
@@ -97,151 +98,74 @@ fail validation.
 			<td>控制（Control）</td>
 			<td>策略（Policy）</td>
 		</tr>
-    <tr>
+    		<tr>
```

Comment:
The original file is already mixed spaces and tabs :(

Reply:
Tabs should be removed. It warrants another PR.